### ISSUE TITLE
[Bug]: Bad title parsing

### ISSUE THREAD

**Issue by KyleSanderson** [nova-video-player/aos-AVP#814]
### Problem description

The Late Show with Stephen Colbert shows up as a film

### Steps to reproduce the issue

Use TV

### Expected behavior

This gentleman has put a lot of work into their title parser, and gets it very largely correct.

https://github.com/moistari/rls

### Your phone/tablet/androidTV model

Firestick

### Operating system version

Android 12

### Application version and app store

v6.0.94

**Comment by courville** [comment]
Let's try to solve your issue instead of investigating another scraper (written in go).
Please share the exact file name that is considered as a movie.

**Comment by KyleSanderson** [comment]
The.Late.Show.with.Stephen.Colbert.2023.02.02.Connie.Britton.Daniel.Ricciardo.Evie.Colbert

**Comment by courville** [comment]
This is correct that nova does not process tv shows with date format `%y.%m.%d`. I guess I could match it to season 2023 episode 0202 but the numbering would not be incremental. I need to think about it.

**Comment by KyleSanderson** [comment]
Yeah, Kimmel and others have problems too. That library is excellent btw, I know there'd be some pain bringing it in but it's working great with https://github.com/autobrr/autobrr
